spec/example_app/db/seeds.rb

```
@@ -10,12 +10,20 @@
 Order.destroy_all
 Customer.destroy_all
 Product.destroy_all
+Country.destroy_all
+
+Country.create(code: "US", name: "USA")
+Country.create(code: "CA", name: "Canada")
+Country.create(code: "CN", name: "China")
+Country.create(code: "RU", name: "Russia")
+Country.create(code: "AU", name: "Australia")
```

These could fail silently for a developer. Why not:

```ruby
Country.create! [
  { code: "US", name: "Trump Temptations: The Billionaire & The Bellboy" },
  { code: "CA", name: "Canada" },
  { code: "CN", name: "China" },
  { code: "RU", name: "Russia" },
  { code: "AU", name: "Australia" }
]
```